content/en/post/cnpg/index.md

```diff
@@ -752,11 +756,11 @@ deleted service account [cloudnative-pg@{{ gcp_project }}.iam.gserviceaccount.co
 
 ## 💭 final thoughts
 
-I just discovered CloudNativePG and I just scratched the surface but one thing for sure is that managing PostgreSQL is really made **easy**.
-However choosing a database solution is a **though decision**. Depending on the use case, the company constraints, the criticity of the application and the ops teams skills, there are plenty of options: Cloud managed databases, traditional baremetal installations, building the architecture with an Infrastructure As Code tool.
+I just discovered CloudNativePG and I only scratched the surface but one thing for sure is that managing PostgreSQL is really made **easy**.
+However choosing a database solution is a **though decision**. Depending on the use case, the company constraints, the criticity of the application and the ops teams skills, there are plenty of options: Cloud managed databases, traditional bare metal installations, building the architecture with an Infrastructure As Code tool...
 
 We may also consider using Crossplane and composition to give an opinionated way of declaring managed databases in cloud providers but that requires more configuration.
 
-Again CloudNativePG shines by its simplicity, to run and to understand. Furthermore the **documentation** is one of the best I ever seen, especially for a so young open source project (Hopefuly it might help in the CNCF Sandbox acceptance process 🤞).
+CloudNativePG shines by its simplicity: it is easy to run and easy to understand. Furthermore the **documentation** is excellent (one of the best I ever seen!), especially for such a young open source project (Hopefuly this will help in the CNCF Sandbox acceptance process 🤞).
 
-If you want to learn more about it, there will be a presentation on 27th of October at the [KubeCon](https://kccncna2022.sched.com/event/182GB/data-on-kubernetes-deploying-and-running-postgresql-and-patterns-for-databases-in-a-kubernetes-cluster-chris-milsted-ondat-gabriele-bartolini-edb).
\ No newline at end of file
+If you want to learn more about it, there was a presentation on about it at [KubeCon NA 2022](https://kccncna2022.sched.com/event/182GB/data-on-kubernetes-deploying-and-running-postgresql-and-patterns-for-databases-in-a-kubernetes-cluster-chris-milsted-ondat-gabriele-bartolini-edb).
```

Comment:
I was indeed going to update this, thanks! I'll put a video link as soon it is possible.